classes/classes/Player.as

```diff
@@ -1653,13 +1653,22 @@ use namespace CoC;
 		//Is Bows
 		public function isBowTypeWeapon():Boolean
 		{
-			return weaponRangePerk == "Bow";
+			return weaponRangePerk == ItemConstants.WT_BOW;
+		}
+		//Is Crossbow
+		public function isCrossbowTypeWeapon():Boolean
+		{
+			return weaponRangePerk == ItemConstants.WT_CROSSBOW;
 		}
 		//Is Thrown
 		public function isThrownTypeWeapon():Boolean
 		{
 			return weaponRangePerk == "Throwing";

```

Comment:
change this to WT_THROWING for consistency sake

Reply:
This has been changed to use the constant